src/app/core/models/rules-interfaces.model.ts

```diff
@@ -6,6 +6,7 @@ export class Rule{
   deleted_tardiness_rules?: TardinessRule [] = [];
   leave_type_id?: number;
   is_half_day?: boolean;
+  time_groups?:boolean;
```

Comment:
I think we need time_groups to be string not boolean, as its retrieved as a string

Reply:
done